src/App.tsx

```diff
@@ -1,26 +1,112 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { addTodo, deleteTodo, getTodos, USER_ID } from './api/todos';
+import { Header } from './components/header';
+import { TodoList } from './components/todoList';
+import { Footer } from './components/footer';
+import { Notification } from './components/notification';
+import { Todo } from './types/Todo';
+import { Completed } from './types/Filters';
+import { handlefilterTodos } from './utils/filterTodos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filtersParams, setFiltersParams] = useState(Completed.All);
+  const [messageError, setMessageError] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState(NaN);
+  const [isDelComplited, setDelComplited] = useState(false);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => setMessageError('Unable to load todos'));
+  }, []);
+
+  useEffect(() => {
+    const timer = setTimeout(() => setMessageError(''), 3000);
+
+    return () => clearTimeout(timer);
+  }, [messageError]);
+
+  const addNewTodo = (newTodo: Todo): Promise<Todo | void> => {
+    return addTodo(newTodo).then(todo => {
+      setTodos(prevTodos => [...prevTodos, todo]);
+    });
+  };
```

Comment:
shouldn't there be also a catch section in case of error? Like 'Unable to add a todo'' or something?